client/verta/verta/monitoring/summaries/queries.py

```diff
@@ -234,6 +239,23 @@ def _find_summaries(self, proto_summary_query):
         summary_query = SummaryQuery._from_proto_request(proto_summary_query)
         self._summary_query = summary_query
 
+    @property
+    def aggregation(self):
+        return self._aggregation
+
+    @aggregation.setter
+    def aggregation(self, value):
+        if value is None:
+            self._aggregation = None
+        elif isinstance(value, AggregationQuerySummary):
+            self._aggregation = Aggregation._from_proto(value)
+        elif isinstance(value, Aggregation):
+            self._aggregation = value
+        else:
+            raise ValueError(
+                "Not a valid aggregation type"
```

Comment:
Maybe something like this
```suggestion
                "value must be Aggregation object or proto, not {}".format(type(value))
```